### DISCUSSION TITLE
본인인증에서 자동 예약 진행하는 거 맞나요??

### DISCUSSION THREAD

**Discussion by DlgurWwkd** [changdoc/naver-vaccine-macro#162] [Q&A]
예약신청으로 넘어가지 않습니다.
어떻게 해야하나요?
![image](https://user-images.githubusercontent.com/88217232/127724702-8f520206-557a-48b4-9f8f-a64e79fa49a7.png)

**Comment by changdoc** [comment]
먼저 본인인증을 하셔야 합니다

**Reply by DlgurWwkd** [reply]
org랑  id 그대로 입력하고 들어가서 인증을 했는데도 다음으로 넘어가지 않네요ㅜ

**Reply by DlgurWwkd** [reply]
![image](https://user-images.githubusercontent.com/88217232/127726843-849dca6d-143d-4f7f-9132-df415ccafce1.png)
주소창을 보니 info? 가 아니라 auto?로 돼있는데 문제있는거 맞나요?

**Reply by DlgurWwkd** [reply]
그냥 네이버가 문제였네요 확장프로그램 삭제하고 다시 해봐도 넘어가질 않네요
